docs/tutorials/tutorial-query.md

```diff
@@ -216,7 +216,13 @@ Retrieved 10 rows in 0.06s.
 ### Query SQL over HTTP
 
 
-You can submit queries directly to the Druid Broker over HTTP. 
+You can submit native queries [directly to the Druid Broker over HTTP](../querying/sql.md#http-post). The request body should be a JSON object, with the value for the key `query` containing text of the query:
+
+```json
+{
+  "query": "SELECT ..."
```

Comment:
I think a complete SQL, such as the SQL example in above section, is better.

Reply:
Good idea, Frank. To offer as a suggestion the SQL in `wikipedia-top-pages-sql` mentioned in the paragraph that follows this one: 

```suggestion
  "query": "SELECT page, COUNT(*) AS Edits FROM wikipedia WHERE \"__time\" BETWEEN TIMESTAMP '2015-09-12 00:00:00' AND TIMESTAMP '2015-09-13 00:00:00' GROUP BY page ORDER BY Edits DESC LIMIT 10"
```

Reply:
👍